site2/docs/developer-landing.md

```diff
@@ -0,0 +1,17 @@
+---
+Id: developers-landing

```

Comment:
@DaveDuggins this file is duplicate. We've got [this landing page for developers](https://pulsar.apache.org/docs/next/developers-landing/) online already.

Reply:
The new draft of the page has changes, so it's fine if it overwrites the old draft.